tests/DocBlock/TypeExpressionTest.php

```diff
@@ -358,6 +370,8 @@ public static function provideParseInvalidExceptionCases(): iterable
         yield ['\' unclosed string'];
 
         yield ['\' unclosed string \\\''];
+
+        yield 'generic with no arguments' => ['f<>'];
```

Comment:
What is the point of generic without arguments?

Reply:
Exactly, no point, thus not allowed ;-)